release/release_after_kokoro.sh

```diff
@@ -230,7 +230,7 @@ echo "THIS WILL DELETE ALL FILES IN '$DESTINATION' IF THEY EXIST!"
 ask_proceed
 
 set -x
-gsutil -m rm $DESTINATION/** && \
+gsutil -m rm $DESTINATION/** || true
```

Comment:
Do we need this `|| true`?  We're not using `set -o errexit` that I can see.

Reply:
Good catch!